src/valkey_search.cc

```
@@ -143,7 +91,11 @@ void ValkeySearch::InitInstance(std::unique_ptr<ValkeySearch> instance) {
 }
 
 uint32_t ValkeySearch::GetHNSWBlockSize() const {
-  return hnsw_block_size.load(std::memory_order_relaxed);
+  return Options::Instance().GetHNSWBlockSize().GetValue();
+}
+
+void ValkeySearch::SetHNSWBlockSize(uint32_t block_size) {
+  Options::Instance().GetHNSWBlockSize().SetValue(block_size).IgnoreError();
```

Here and other places, I don't see a reason why we would want to ignore errors, at minimum we should log those.

---

Fixed